packages/vtx-react-scripts/src/commands/test.ts

```
@@ -0,0 +1,30 @@
+import { Command, flags } from '@oclif/command';
+import { runCLI } from 'jest';
+import { load } from 'npm';
+import * as path from 'path';
+
+export default class Test extends Command {
+  static description = 'Runs tests';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+  };
+
+  async run() {
+    this.log('Running tests...');
+
+    load(() => {
+      const options = {
+        rootDir: process.cwd(),
+        passWithNoTests: true,
+      };
+      runCLI(options as any, [path.resolve(process.cwd(), './jest.config.js')])
+        .then((success) => {
+          console.log(success.results);
+        })
+        .catch((failure) => {
+          console.error(failure);
```

Again, the process must be a failing process ID. So it is counterproductive and would create a false positive to swallow an error. Instead don’t try catch at all and don’t .catch

---

Removed